app/models/subscription.rb

```diff
@@ -62,4 +62,8 @@ def available_for_canceling?
   def permitted_payment_methods
     plan.payment_methods
   end
+
+  def must_have_cpf?
```

Comment:
I think this method name should be more descriptive to _explicitly_ show the business rule here. What do you think about `from_brazil?` or `portuguese_language?`?